tests/test_api.py

```diff
@@ -1812,10 +1823,10 @@ def test_reactor_config_secret(self):
         ("plat", 'rel', None, True, False, 1, False),
         ("plat", 'rel', None, True, False, None, True),
         # orchestrator build
-        (None, None, 'platforms', False, True, 1, True),
+        (None, None, 'platforms', False, True, 1, False),
```

Comment:
As @rcerven pointed out on IRC, this should not fail now because the CLI check incorrectly prohibited use of `arrangement_version` for orchestrator builds.